jax-maven-plugin-test/pom.xml

```diff
@@ -56,6 +56,31 @@
                     <target>${maven.compiler.target}</target>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <version>1.6</version>
+                <executions>
+                    <execution>
+                        <id>copy-schema-src</id>
+                        <phase>initialize</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <echo>Copying code for the schemagen with source option test</echo>
+                                <mkdir dir="${project.build.directory}/schema-src"/>
+                                <copy todir="${project.build.directory}/schema-src">
+                                    <fileset dir="${basedir}/../jax-maven-plugin-test-jaxb/src/main/java">
```

Comment:
Why not reference this path directly in the source element?

Reply:
@davidmoten good point :) updated